### PR TITLE
refactor(error-handler): remove the irritating DM message

### PR DESCRIPTION
**Pull request by spifory**
This pull request removes the thing where it sends a DM to all 3 of the devs when there's a command error. It is very useless currently as there are no current actual errors, it's sometimes just people purposely doing this.
**Note:** This has not been tested.

### CODE REVIEW

**Review by ThePrivatePanda — APPROVED** [review]
Finolly!

**Review by ooliver1 — APPROVED** [review]

**Comment by ooliver1** [comment]
:trollface:

File: bolb_bot/cogs/events.py [hunk 1]
```diff
@@ -68,14 +68,6 @@ async def on_command_error(self, ctx: Context, error: Exception):
                 color=self.bot.color,
             )
             ensure_future(ctx.send(embed=embed))
-            if ctx.guild is None:
```
Comment (ooliver1) [inline bolb_bot/cogs/events.py:71]: May want to remove "dev has been notified" from above

**Review by ooliver1 — CHANGES_REQUESTED** [review]

File: bolb_bot/cogs/events.py (hunk 1 above)
Comment (spifory) [reply bolb_bot/cogs/events.py:71]: > May want to remove "dev has been notified" from above

It does technically log the error and command to the console though, but what do you say?

**Pull request merged**